node/libs/roles/src/attester/keys/secret_key.rs

```diff
@@ -36,7 +36,11 @@ impl SecretKey {
 
     /// Sign a message hash.
     pub fn sign_hash(&self, msg_hash: &MsgHash) -> Signature {
-        Signature(self.0.sign(&ByteFmt::encode(msg_hash)))
+        let sig = self
+            .0
+            .sign_hash(&ByteFmt::encode(msg_hash))
+            .expect("message hash is sufficient in size");
```

Comment:
you mean "has insufficient size"?

Reply:
I mean I know it has sufficient size, which is why I expect that this will not fail, but I see how it is confusing, will change it.